operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -21,6 +21,11 @@ kiali_defaults:
       - "kiali-operator"
       #label_selector:
 
+  apidocs:
+    annotations:
+      api_type_annotation_name: "kiali.kubernetes.io/api-type"
```

Comment:
trivial change request - all peer settings should be alphabetically sorted (as they already are in your comments up above on lines 75-76), so just swap the type and spec setting here to maintain alphabetical ordering
```
  apidocs:
    annotations:
      api_spec_annotation_name: "kiali.kubernetes.io/api-spec"
      api_type_annotation_name: "kiali.kubernetes.io/api-type"
```